.npmignore

```diff
@@ -2,11 +2,11 @@
 src
 babel.config.js
 .eslintrc
+.eslintignore
```

Comment:
Seems like this should be commited separately?

Reply:
moved to commit c82b3ffbba4fe08e700ea9ee75568c0895411351

Reply:
Thanks!